src/lib.rs

```diff
@@ -485,8 +485,8 @@ impl SessionOptions {
 
     /// Set the config.
     ///
-    /// `config` should be a serialized brain.ConfigProto proto.
-    /// Returns an error if config was not parsed successfully as a ConfigProto.
+    /// `config` should be a serialized [`ConfigProto` proto](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/protobuf/config.proto).
+    /// Returns an error if config was not parsed successfully as a `ConfigProto`.
```

Comment:
Seemed like a good idea to actually go ahead and get that link in place as I had to go looking for that information. Let me know if you thinking it would be best to pin to a specific release version as opposed to just master.

Reply:
master is fine.  If we pinned it to a specific version, it'd fall out of sync with the version we pull in.